src/inspector_agent.cc

```diff
@@ -449,7 +452,9 @@ Agent::Agent(Environment* env) : parent_env_(env),
                                  client_(nullptr),
                                  platform_(nullptr),
                                  enabled_(false),
-                                 next_context_number_(1) {}
+                                 next_context_number_(1),
```

Comment:
Question: any reason do keep using initialization lists when we support C++11 (i.e. "Class Member Initialization")?